### PR TITLE
Add RTL style to setup wizard

### PR DESCRIPTION
**Pull request by kadimi**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changelog entry

Add RTL style to setup wizard.

### CODE REVIEW

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -205,6 +205,7 @@ public function enqueue_scripts() {
 		wp_enqueue_style( 'wc-setup', WC()->plugin_url() . '/assets/css/wc-setup.css', array( 'dashicons', 'install' ), WC_VERSION );
 
 		wp_register_script( 'wc-setup', WC()->plugin_url() . '/assets/js/admin/wc-setup' . $suffix . '.js', array( 'jquery', 'wc-enhanced-select', 'jquery-blockui', 'wp-util', 'jquery-tiptip' ), WC_VERSION );
+		wp_style_add_data( 'wc-setup', 'rtl', 'replace' );
```
Comment (rodrigoprimo) [inline includes/admin/class-wc-admin-setup-wizard.php]: Could you please move this line up and place it right after `wc-setup` style (and not the script) is enqueued?

**Review by rodrigoprimo — COMMENTED** [review]
Thanks for your contribution and I'm sorry for the delay to respond, @kadimi. I was able to test this change and confirm that it works as expected.

Unfortunately, I'm not able to merge it due to a conflict, and for some reason, I couldn't fix it myself without losing the reference to your commit. Could you please fix the conflict?

I also left one inline comment with a small suggestion.

**Comment by peterfabian** [comment]
Hi @kadimi , 

Would you be able to fix the suggestion from @rodrigoprimo ? As we can't cherry pick your commit, we can only add the fix ourselves, but we don't want to remove your contribution.

Thanks!

**Comment by kadimi** [comment]
Thanks @rodrigoprimo and @peterfabian. I will try to find some free time to work on this one this week.

**Comment by rodrigoprimo** [comment]
Closing in favor of #25835.

**Pull request closed without merging**